pykylin/cursor.py

```diff
@@ -17,15 +24,51 @@ def __init__(self, connection):
         self.fetched_rows = 0
 
     def callproc(self):
-        raise('Stored procedures not supported in Kylin')
+        raise ('Stored procedures not supported in Kylin')
 
     def close(self):
         logger.debug('Cursor close called')
 
+    @staticmethod
+    def trans_sql_for_kylin(sql_str):
```

Comment:
All these tricky sql transform steps are because Caravel does't support multi-table join directly.
So we put the multi-table join transform into pykylin as a simple work around.
We have plan to modify Caravel for multi-table join support. The code here will be deleted in the future.